pages/common/psql.md

```diff
@@ -2,22 +2,22 @@
 
 > PostgreSQL command-line client.
 
-- Connect to *database*. It connects to localhost using default port *5432* with default user:
+- Connect to database. It connects to localhost using default port 5432 with default user:
 
 `psql {{database}}`
 
-- Connect to *database* on given server *host* running on given *port* with *username* given, no password prompt:
+- Connect to database on given server host running on given port with username given, no password prompt:
 
 `psql -h {{host}} -p {{port}} -U {{username}} {{database}}`
 
-- Connect to *database*, user will be prompted for password:
+- Connect to database, user will be prompted for password:
```

Comment:
use a semicolon [instead of a comma](https://en.wikipedia.org/wiki/Run-on_sentence), since these are separate clauses.